rules/config

```diff
@@ -126,7 +126,7 @@ DEFAULT_VS_PREPARE_MEM = yes
 
 
 # INCLUDE_SYSTEM_EVENTD - build docker-eventd for system eventd support
-INCLUDE_SYSTEM_EVENTD = y
+INCLUDE_SYSTEM_EVENTD = n
```

Comment:
Offline discussed, there is no fine-granularity switch to disable and unblock. So we made the decision to disable structure event feature.